tests/dags/providers/provider_api_scripts/test_inaturalist.py

```diff
@@ -107,3 +107,39 @@ def test_get_media_type():
     expected = "image"
     actual = INAT.get_media_type({"some test": "data"})
     assert actual == expected
+
+
+ti_mock = mock.MagicMock(spec=TaskInstance)
```

Comment:
Probably best to have this as a fixture so it's recreated on every test! If we wanted to check call count or anything, we'd get results from all the previous tests that used this object too.

Alternatively, since you're just using it in the test below, you can probably instantiate it within the test and remove it from the parametrizations 🙂